erizo/src/erizo/WebRtcConnection.cpp

```diff
@@ -201,8 +211,10 @@ bool WebRtcConnection::processRemoteSdp() {
   extension_processor_.setSdpInfo(local_sdp_);
   local_sdp_->updateSupportedExtensionMap(extension_processor_.getSupportedExtensionMap());
 
-  local_sdp_->video_ssrc_list.push_back(media_stream_->getVideoSinkSSRC());
-  local_sdp_->audio_ssrc = media_stream_->getAudioSinkSSRC();
+  forEachMediaStream([this] (const std::shared_ptr<MediaStream> &media_stream) {
+    local_sdp_->video_ssrc_list.push_back(media_stream->getVideoSinkSSRC());
+    local_sdp_->audio_ssrc = media_stream->getAudioSinkSSRC();
```

Comment:
This is not part of this PR but, do you have a plan here? The obvious solution is to have another list for audio ssrcs.

Reply:
I'd rather move the ssrc tracking to a new class called stream_sdp or similar.